packages/core/src/lib/select/select.svelte

```diff
@@ -25,9 +43,35 @@ export let value: string | undefined = undefined;
 export let disabled = false;
 
 /** The state of the select (info, warn, error, success), if any. */
-export let state: InputState = 'none';
+export let state: SelectState = 'none';
+
+const onInput = (event: Event) => {
```

Comment:
Why are the disabled checks necessary here? Shouldn't native selects already handle disabling input when disabled?

I'm primarily worried about how this degrades the experience of event forwarding, since these event objects are being wrapped in the custom event detail attribute.

If we end up finding this pattern necessary - ie intercepting events before forwarding, I think it would be valuable to replicate Threlte's [createRawEventDispatcher](https://threlte.xyz/docs/reference/core/utilities#createraweventdispatcher) in PRIME so that we can forward raw event objects.

Reply:
Because disabled inputs and selects are not focusable/keyboard navigable and completely hidden to screen readers.

Reply:
🤦 Oh yes, keep forgetting about this problem. Then based on this and other discussions I've observed I'd recommend we replicate [createRawEventDispatcher](https://threlte.xyz/docs/reference/core/utilities#createraweventdispatcher) and maybe [forwardEventHandlers](https://threlte.xyz/docs/reference/core/utilities#forwardeventhandlers), I'd hate for us to end up back in the react space of having things like `nativeEvent` properties :(

Reply:
So I looked at how Threlte did it and they rely on `svelte/internal` which was hidden form the public API and is likely to change in v5, so I wonder if there is a better solution.

Reply:
Updated per our conversation.